go/mysql/server.go

```diff
@@ -42,10 +42,10 @@ const (
 
 var (
 	// Metrics
-	timings    = stats.NewTimings("MysqlServerTimings")
-	connCount  = stats.NewInt("MysqlServerConnCount")
-	connAccept = stats.NewInt("MysqlServerConnAccepted")
-	connSlow   = stats.NewInt("MysqlServerConnSlow")
+	timings    = stats.NewTimings("MysqlServerTimings", "MySQL server timings")
+	connCount  = stats.NewGauge("MysqlServerConnCount", "Active MySQL server connections")
+	connAccept = stats.NewCounter("MysqlServerConnAccepted", "Count of connections accepted by MySQL server")
```

Comment:
Admittedly this is taste, but I find the use of "Count of" to be unnecessary in this and other descriptions. Ideally we should be consistent one way or the other and either always use it or never use it.

Personally I find "Accepted MySQL server connections" to be sufficiently descriptive and more consistent with the gauge case (where we don't say "Gauge of active MySQL server connections").